docs/dictionary/command/stop-session.lcdoc

```diff
@@ -20,6 +20,6 @@ Use the <stop session> command to end the session explicitly and save the conten
 
 The location where the $_SESSION array is stored is configured using the <sessionSavePath property>.
```

Comment:
@marksmithhfx You'll need to change the reference here to `<sessionSavePath> property` so that it links correctly.